packages/web/pages/user/my-account/index.js

```diff
@@ -172,87 +184,91 @@ const CommonDataForm = ({ form, user, message, loading }) => {
 						placeholder={t('account:placeholders.birthDate')}
 						mask="99/99/9999"
 						pattern={/^([0-2][0-9]|(3)[0-1])(\/)(((0)[0-9])|((1)[0-2]))(\/)\d{4}$/i}
+						variant="gray"
 					/>
 				</Cell>
-				<Cell col={2}>
-					<InputField
+				<Cell col={4}>
+					<MaskedInputField
 						form={form}
 						name="phone_number"
 						label={t('account:labels.phoneNumber')}
 						defaultValue={user?.phone_number ?? ''}
 						placeholder={t('account:placeholders.phoneNumber')}
-					/>
-				</Cell>
-				<Cell col={2}>
-					<InputField
-						form={form}
-						name="lattes_id"
-						type="number"
-						label={t('account:labels.lattesId')}
-						defaultValue={user?.lattes_id ?? ''}
-						placeholder={t('account:placeholders.lattesId')}
+						mask="(99) 99999-9999"
```

Comment:
The mask here has an incorrect pattern, it should be `(99) 9999-99999`. It would also look better if `maskChar` is set to `null`:
![Peek 2020-12-09 20-27](https://user-images.githubusercontent.com/10481028/101701143-11353000-3a5d-11eb-99b0-db525b2eeb7b.gif)